### PR TITLE
Added primary string variable in app setting.json. Updated Readme

### PR DESCRIPTION
**Pull request by YukiKitaoka**
3002 in the Amido stacks cycle 6. 

#### 📲 What

Moved the cosmos db primary string into appsettings.json

#### 🤔 Why
It's not a good practice to put a hard coded value into the code.

#### 🕵️ How to test

Notes for QA
Ran all tests locally and passed. 
#### ✅ Acceptance criteria Checklist
- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: src/Amido.Stacks.Data.Documents.CosmosDB.Tests/appsettings.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+    "DatabaseAccountUri": "https://localhost:8081/",
+    "DatabaseName": "Stacks",
+    "DatabasePrimaryKey": "",
+    "SecurityKeySecret": {
+      "Identifier": "COSMOSDB_KEY",
+      "Source": "Environment"
     }
-}
+  }
+}
```
Comment (ElvenSpellmaker) [inline src/Amido.Stacks.Data.Documents.CosmosDB.Tests/appsettings.json:11]: Files should end in a newline =)

File: src/Amido.Stacks.Data.Documents.CosmosDB.Tests/Integration/IntegrationFixture.cs [hunk 2]
```diff
@@ -33,8 +33,9 @@ public async Task InitializeAsync()
                 if (Settings.DatabaseAccountUri.Contains("localhost", StringComparison.InvariantCultureIgnoreCase) &&
                     Environment.GetEnvironmentVariable("VisualStudioEdition") != null)
                 {
+                    var primaryString = this.Settings.DatabasePrimaryKey;
                     Environment.SetEnvironmentVariable(Settings.SecurityKeySecret.Identifier,
-                        "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==");
+                        primaryString);
```
Comment (ElvenSpellmaker) [inline src/Amido.Stacks.Data.Documents.CosmosDB.Tests/Integration/IntegrationFixture.cs:38]: Is `primaryKey` a better variable name than `...String`?

Comment (YukiKitaoka) [reply src/Amido.Stacks.Data.Documents.CosmosDB.Tests/Integration/IntegrationFixture.cs]: Updated per feedback.

File: src/Amido.Stacks.Data.Documents.CosmosDB.Tests/appsettings.json [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
-        "DatabaseAccountUri": "https://localhost:8081/",
-        "DatabaseName": "Stacks",
-        "SecurityKeySecret": {
-            "Identifier": "COSMOSDB_KEY",
-            "Source": "Environment"
-        }
+  "CosmosDb": {
+    "DatabaseAccountUri": "https://localhost:8081/",
+    "DatabaseName": "Stacks",
+    "PrimaryKey": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
```
Comment (underscoreHao) [inline src/Amido.Stacks.Data.Documents.CosmosDB.Tests/appsettings.json:5]: We shouldn't have a primary key exposed like that

**Review by underscoreHao — CHANGES_REQUESTED** [review]

**Review by underscoreHao — APPROVED** [review]

**Review by scottvaughanamido — APPROVED** [review]

**Pull request merged**